clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/cache/Keys.groovy

```diff
@@ -61,7 +61,7 @@ class Keys {
         break
       case Namespace.CLUSTERS.ns:
         def names = Names.parseName(parts[4])
-        result << [application: parts[3], account: parts[2], name: parts[4], stack: names.stack, detail: names.detail]
+        result << [application: parts[3], account: parts[2], name: parts[4], cluster: parts[4], stack: names.stack, detail: names.detail]
```

Comment:
Needed since the `urlMappingTemplate` for `CLUSTERS` in `AwsProvider` refers to `$cluster`.